cognite/client/_api/functions.py

```diff
@@ -525,6 +523,29 @@ def convert_file_path_to_module_path(file_path: str) -> str:
     return ".".join(Path(file_path).with_suffix("").parts)
 
 
+def _get_handle_function_node(handler_path: Path) -> ast.FunctionDef | None:
```

Comment:
Is the `|` typing okay here? I am developing on Python 3.10.6. Ruff auto-corrected it to `|` from `Optional`.